scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -283,45 +355,87 @@ task CalculateTargetCoverage {
     Boolean keep_duplicate_reads = true
     Boolean disable_all_read_filters = false
     File input_bam
+    File bam_idx
     File ref_fasta
     File ref_fasta_fai
     File ref_fasta_dict
     File gatk_jar
     String is_disable_reference_validation
+    Boolean isWGS
     Int mem
 
+    # Note that when isWGS is true, this task is still called by the workflow.
+    # In that case, to satisfy the output requirements an empty coverage file is created
```

Comment:
In that case, to satisfy the output requirements, an empty coverage file is created